firmware/application/apps/capture_app.cpp

```diff
@@ -86,7 +86,7 @@ CaptureAppView::CaptureAppView(NavigationView& nav)
     };
 
     receiver_model.enable();
-    option_bandwidth.set_selected_index(7);  // Preselected default option 500kHz.
+    option_bandwidth.set_selected_index(4);  // Preselected default option 500kHz.
```

Comment:
FYI, you could use this instead, if the preferred bandwidth is 500kHz:  Then it wouldn't matter what the index value is:
`option_bandwidth.set_by_value(500000);`